src/domain/journey/common/AboutPageContainer/AboutPageContainer.tsx

```diff
@@ -85,6 +89,12 @@ const AboutPageContainer: FC<AboutPageContainerProps> = ({ journeyId, children }
   });
 
   const memberProfile = membersData?.lookup.space?.profile;
+  const virtualContributors = membersData?.lookup.space?.community.virtualContributors.filter(
+    vc => vc.searchVisibility === SearchVisibility.Public
```

Comment:
We've discussed that filtering briefly. I think we should include the visibility per Account/Private on that level too.
Feel free to leave it like that but please mark it in the ticket.

Reply:
There is an explicit bullet to not show private vcs here. It's not clear what you want here.

Reply:
I'll update the ticket, thanks!